src/mixins/filing-template-mixin.ts

```diff
@@ -1094,20 +1099,20 @@ export default class FilingTemplateMixin extends DateMixin {
     })
 
     // store Folio Number
-    // FUTURE: should we store correction.folioNumber instead?
-    this.setFolioNumber(entitySnapshot?.authInfo?.folioNumber || '')
```

Comment:
Instead of falling back to save the default value (which isn't our job; the defaults should be in one place only), I rewrote this. (Also below.)